src/components/ParticipantList.vue

```diff
@@ -93,7 +93,7 @@ Licensed under the Elastic License 2.0. */
       field: 'studyGroupId',
       header: t('study.props.studyGroup'),
       type: MoreTableFieldType.choice,
-      editable: { values: groupStatuses.value },
+      editable: { values: groupStatuses.value, editable: actionsVisible },
```

Comment:
mir gefällt das naming hier nicht: in dem `editable` Attribut nochmal ein `editable` attribute zu haben. Da braucht es einen sinnvolleren namen

Reply:
Früher war es so, das sobald ein Object mit "values" vorhanden war, war es editable. Nun musste ich noch weiter unterscheiden. Ich habe nun das property "showValuesForEditing" benannt.